plugins/msr/fu-plugin-msr.c

```diff
@@ -370,6 +370,7 @@ fu_plugin_add_security_attr_amd_sme_enabled(FuPlugin *plugin, FuSecurityAttrs *a
 	/* success */
 	fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS);
 	fwupd_security_attr_set_result(attr, FWUPD_SECURITY_ATTR_RESULT_ENCRYPTED);
+	fwupd_security_attr_add_obsolete(attr, "pci_psp");
```

Comment:
I'm not familiar with fwupd_security_attr_add_obsolete so this is merely a request for clarification:
- what happens in practice, is there a race depending on which module is loaded first since they obsolete each other?
- since you mention in the commit message it's not intuitive, worth adding a comment here (and in the other file) to explain what is going on?

Reply:
Just to preface - I still don't like how this works right now, but don't have a better solution.  Hoping we can come up with one in this PR.  You guys don't have hardware that can see this yet, so let me show you what it looks like.

If TSME and SME are disabled you'll see this:
```
HSI-4
✘ Encrypted RAM:                 Invalid
✘ Encrypted RAM:                 Failed
```

With obsoletes If TSME or SME is enabled you see:
```
HSI-4
✔  Encrypted RAM:                 Encrypted
```

If you don't have obsoletes and only one of those is set (what I think is likely to be the more common case for both Ryzen Pro and Epyc) you'll see something like this:
```
HSI-4
✔  Encrypted RAM:                 Encrypted
✘ Encrypted RAM:                 Failed
```

So what happens is if one is set as success then the other doesn't show up.  If both are fail (for whatever reason) then both show up.  We could potentially approach it as dedicated attributes for SME and TSME, but I'm not sure that makes sense either.